website/docs/d/control_plane_ip_addresses.html.markdown

```diff
@@ -0,0 +1,51 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: mongodbatlas_control_plane_ip_addresses"
+sidebar_current: "docs-mongodbatlas-datasource-control-plane-ip-addresses"
+description: |-
+    "Provides a data source that returns all control plane IP addresses"
+---
+
+# Data Source: mongodbatlas_control_plane_ip_addresses
+
+
+Provides a data source that returns all control plane IP addresses.
+
+## Example Usages
+```terraform
+data "mongodbatlas_control_plane_ip_addresses" "test" {
+}
+
+output "outbound-aws-ip-addresses" {
+  value = data.mongodbatlas_control_plane_ip_addresses.test.outbound.aws
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Read-Only
+
+- `inbound` (Attributes) List of inbound IP addresses to the Atlas control plane, categorized by cloud provider. If your application allows outbound HTTP requests only to specific IP addresses, you must allow access to the following IP addresses so that your API requests can reach the Atlas control plane. (see [below for nested schema](#nestedatt--inbound))
+- `outbound` (Attributes) List of outbound IP addresses from the Atlas control plane, categorized by cloud provider. If your network allows inbound HTTP requests only from specific IP addresses, you must allow access from the following IP addresses so that Atlas can communicate with your webhooks and KMS. (see [below for nested schema](#nestedatt--outbound))
+
+<a id="nestedatt--inbound"></a>
+### Nested Schema for `inbound`
+
+Read-Only:
+
+- `aws` (Map of List of String) Control plane IP addresses in AWS. Each key identifies an Amazon Web Services (AWS) region. Each value identifies control plane IP addresses in the AWS region.
+- `azure` (Map of List of String) Control plane IP addresses in Azure. Each key identifies an Azure region. Each value identifies control plane IP addresses in the Azure region.
+- `gcp` (Map of List of String) Control plane IP addresses in GCP. Each key identifies a Google Cloud (GCP) region. Each value identifies control plane IP addresses in the GCP region.
+
+
+<a id="nestedatt--outbound"></a>
+### Nested Schema for `outbound`
+
+Read-Only:
+
+- `aws` (Map of List of String) Control plane IP addresses in AWS. Each key identifies an Amazon Web Services (AWS) region. Each value identifies control plane IP addresses in the AWS region.
+- `azure` (Map of List of String) Control plane IP addresses in Azure. Each key identifies an Azure region. Each value identifies control plane IP addresses in the Azure region.
+- `gcp` (Map of List of String) Control plane IP addresses in GCP. Each key identifies a Google Cloud (GCP) region. Each value identifies control plane IP addresses in the GCP region.
+
+For more information see: [MongoDB Atlas API - Return All Control Plane IP Addresses](https://www.mongodb.com/docs/atlas/reference/api-resources-spec/v2/#tag/Root/operation/returnAllControlPlaneIPAddresses) Documentation.
```

Comment:
```suggestion
For more information see: [MongoDB Atlas API - Return All Control Plane IP Addresses](https://www.mongodb.com/docs/atlas/reference/api-resources-spec/v2/#tag/Root/operation/returnAllControlPlaneIPAddresses) in the Atlas Administration API documentation.
```

Reply:
given that it is already mentioning `MongoDB Atlas API` I believe this additional text could be redundant. WDYT?